app/server/utils/transfer_limits.py

```diff
@@ -33,32 +42,6 @@
 
 GENERAL_PAYMENTS = [STANDARD_PAYMENT, AGENT_OUT_PAYMENT, AGENT_IN_PAYMENT, RECLAMATION_PAYMENT]
 
-def get_transfer_limits(credit_transfer: CreditTransfer):
```

Comment:
just at the bottom of the file now, because it relies on classes already existing